index.bs

```diff
@@ -790,7 +791,10 @@ To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
   1. Let |connection| be the result of [=obtain a connection|obtaining a connection=] with
      |networkPartitionKey|, |url|, false, |newConnection|, and |http3Only|. If the user agent
      supports more than one congestion control algorithm, choose one appropriate for
-     |congestionControl| for sending of data on this |connection|.
+     |congestionControl| for sending of data on this |connection|. When obtaining a connection, if
+     |serverCertificateHashes| is specified instead of the default certificate version algorithm,
+     validate the certificate against [=custom certificate requirements=], and then
+     [=verify a certificate hash|verify the certificate hash=] against |serverCertificateHashes|.
```

Comment:
I would clarify that certificate should be considered valid if and only if _both_ of those checks pass.